dockerfiles/e2e-saas/docker-entrypoint.sh

```diff
@@ -59,13 +59,23 @@ length=${#USERNAME}
 
 echo "Trying to find token for $(echo $USERNAME | cut -c1-3) $(echo $USERNAME | cut -c4-$length)"
 
-#verify environment - if production or prod-preview
-#variable preview is used to differ between prod and prod-preview urls
-rm -rf cookie-file loginfile.html
-if [[ "$USERNAME" == *"preview"* ]] || [[ "$USERNAME" == *"saas"* ]]; then
-  preview="prod-preview."
-else
-  preview=""
+if [ -z $ACCOUNT_ENV ]; then
+  echo "ACCOUNT_ENV variable is not set, retrieving env from URL."
+
+  #verify environment - if production or prod-preview
+  #variable preview is used to differ between prod and prod-preview urls
+  rm -rf cookie-file loginfile.html
+  if [[ "$USERNAME" == *"preview"* ]] || [[ "$USERNAME" == *"saas"* ]]; then
+    preview="prod-preview."
+  else
+    preview=""
+  fi
+else 
+  if [ "$ACCOUNT_ENV" == "prod" ]; then
+    preview=""
+  else 
+    preview="prod-preview."
+  fi
 fi
 
 response=$(curl -s -g -X GET --header 'Accept: application/json' "https://api.${preview}openshift.io/api/users?filter[username]=$USERNAME")
```

Comment:
in general, I'm not a big fan of the manipulations with `${preview}`

Reply:
I believe we can simply determine URLs based on the `ACCOUNT_ENV` and avoid using extra preview env var